vms/platformvm/state/state.go

```diff
@@ -1152,7 +1179,18 @@ func (s *state) loadCurrentValidators() error {
 			return fmt.Errorf("expected tx type txs.Staker but got %T", tx.Unsigned)
 		}
 
-		staker, err := NewCurrentStaker(txID, stakerTx, metadata.PotentialReward)
+		startTime := time.Unix(metadata.StakerStartTime, 0)
+		if metadata.StakerStartTime == 0 {
```

Comment:
regardless what StarTime is specified in AddStakerTx, here StakerStartTime will be the P-chain time at the moment the staker is accepted. Hence it cannot be zero.